source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationType.java

```diff
@@ -28,6 +28,10 @@ public enum CalculationType {
    * Calculate with custom points (fixed set of receptors or user defined points).
    */
   CUSTOM_POINTS,
+  /**
+   *
```

Comment:
Could use a small description as to what the option means (Calculates using custom points mapped to receptors or something like that?)

Reply:
Oh wow how did I miss that, I guess I quickly added this and thought that I would add a description later.